### PR TITLE
Added endpoint and support for pending reviews

### PR DESCRIPTION
**Pull request by fatcat2**

### CODE REVIEW

File: biit_server/meeting_handler.py [hunk 1]
```diff
@@ -54,11 +55,25 @@ def meeting_post(request, auth):
     )
 
     try:
-        meeting_db.add(meeting, id=random_id)
+        meeting_db.add(meeting.to_dict(), id=random_id)
+        print(random_id)
```
Comment (stephend017) [inline biit_server/meeting_handler.py]: remove this or replace it with a proper logging / discord print

File: biit_server/rating_handler.py [hunk 2]
```diff
@@ -96,3 +148,49 @@ def rating_get(request, auth):
...
+    print(rating_db_response)
+
+    ratings = [
+        Rating(document_snapshot=rating_snapshot)
+        for rating_snapshot in rating_db_response
+    ]
+    
+
+    ratings  = [
+        rating.to_dict() for rating in ratings if args["email"] in rating.rating_dict and rating.rating_dict[args["email"]] == -1
```
Comment (stephend017) [inline biit_server/rating_handler.py]: can you add a comment for what this line does, its a beefy boi

**Review by stephend017 — APPROVED** [review]
Looks good, just make the changes I suggested

File: biit_server/meeting_handler.py [hunk 3]
```diff
@@ -54,11 +55,25 @@ def meeting_post(request, auth):
...
 
+    rating_db = Database("ratings")
+    rating = Rating(
+        meeting_id=random_id,
+        rating_dict={user: -1 for user in body["user_list"]}
+    )
+
+    try:
+        rating_db.add(rating.to_dict(), id=random_id)
+        print("success")
```
Comment (lzhahn) [inline biit_server/meeting_handler.py:70]: Remove this or add logging on success

**Review by lzhahn — APPROVED** [review]
LGTM, check out the comments before sending it

**Pull request merged**